src/main/kotlin/ua/marchenko/artauction/common/annotation/profiling/beanPostProcessor/CustomProfilingAnnotationBeanPostProcessor.kt

```diff
@@ -0,0 +1,57 @@
+package ua.marchenko.artauction.common.annotation.profiling.beanPostProcessor
+
+import java.lang.reflect.Method
+import java.lang.reflect.Proxy
+import java.util.concurrent.TimeUnit
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.config.BeanPostProcessor
+import org.springframework.stereotype.Component
+import ua.marchenko.artauction.common.annotation.profiling.annotation.CustomProfiling
+import ua.marchenko.artauction.common.annotation.scheduled.beanPostProcessor.CustomScheduledAnnotationBeanPostProcessor
+
+@Component
+class CustomProfilingAnnotationBeanPostProcessor : BeanPostProcessor {
+
+    private val annotatedMethods: MutableMap<String, Map<Method, TimeUnit>> = mutableMapOf()
```

Comment:
Map<..., Map<..., ...>> looks bad, imho

Reply:
rewrite to MutableMap<String, List<Method>> (bean name + list of annotated methods)